### ISSUE TITLE
WCAG 2.1 issues

### ISSUE THREAD

**Issue by alfredocentinaro** [italia/design-wordpress-theme#70]
Sto testando sia il mio sito iisteramo.edu.it, che il tuo di esempio su GitHub con il validatore WCAG 2.1 https://mauve.isti.cnr.it
Ho riscontrato questi errori comuni che affliggono entrambi i siti:
 SC 1.4.10 - Tech SCR34 - Calculating size and position in a way that scales with text size 
 SC 1.3.4 - Tech F97 - Failure due to locking the orientation to landscape or portrait view
C'è modo di correggerli facilmente o valutare un fix ufficiale?
Grazie mille

**Comment by angelavit** [comment]
Ciao,
per le verifiche di accessibilità ho utilizzato Mauve++ e Siteimprove, entrambi rilevano, per il Widget Ricerca, queste violazioni:

2.4.7 Focus Visible
3.3.2 Labels or Instructions

è risolvibile?

grazie
Angela

**Comment by marcogargano** [comment]
Ciao @angelavit 
nel tema la ricerca è quella nativa di WordPress, non saprei a cosa possa esser riferito; bisognerebbe avere più informazioni. 
Strumenti automatici di questo genere ne esistono tanti, poi bisogna sempre verificare anche i falsi positivi.
